card/lib/cardio.rb

```diff
@@ -76,8 +77,22 @@ def set_paths paths
       add_path "db/seeds", :with => "db/seeds.rb"
 
       add_path 'config/initializers',  :glob => '**/*.rb'
-      paths['config/initializers'] << "#{gem_root}/mod/**{,/*/**}/initializers"
-      paths['config/initializers'] << "#{root}/mod/**{,/*/**}/initializers"
+
+    end
+
```

Comment:
afaik, nobody needs custom paths for mod directories now; just seems likely at some point.  The current code won't yet support that (at least for initializers).